internal/store/postgres/asset_repository.go

```diff
@@ -431,6 +431,29 @@ func (r *AssetRepository) GetProbesWithFilter(ctx context.Context, flt asset.Pro
 	return results, nil
 }
 
+func (r *AssetRepository) GetAssetDataAttributes(ctx context.Context, urns []string) (map[string]asset.Asset, error) {
+	stmt := sq.Select(`
+			id, urn, name, type, data->'attributes' as data
```

Comment:
Can you elaborate on the use case for selectively fetching `data.attributes`? Not 100% sure if it makes sense outside of particular use cases.

Reply:
Yes.  In our Frontend UI which calls the lineage API - we are planning to show some filters based on attributes being set through the `enrich` plugin in meteor. With these attributes, we plan to do some grouping of nodes/ filtering if necessary. 

I understand that this might be a very specific use case but given the user customizable attributes in this data field, I think this might be helpful in a broad sense also, adding props to nodes in lineage graph.